server/services/store/sqlstore/user.go

```diff
@@ -158,3 +158,29 @@ func (s *SQLStore) UpdateUserPasswordByID(userID, password string) error {
 
 	return nil
 }
+
+func (s *SQLStore) GetUsersByWorkspace(workspaceID string) ([]*model.User, error) {
+	// TODO: implement this to fetch users from Mattermost
```

Comment:
Nope, that is wrong, you have to implement this to get the users from Focalboard users table, then I going to put a "layer" on top of that, to get the users from mattermost database whenever this is running as a plugin.

Reply:
Done. I'm now fetching users from Focalboard `users` table.